tests/system/feature_store/test_feature_store.py

```diff
@@ -125,7 +126,10 @@ def _generate_random_name():
     return random_name
 
 
-kafka_brokers = os.getenv("MLRUN_SYSTEM_TESTS_KAFKA_BROKERS")
+test_environment = TestMLRunSystem._get_env_from_file()
+kafka_brokers = test_environment["MLRUN_SYSTEM_TESTS_KAFKA_BROKERS"] or os.getenv(
```

Comment:
```suggestion
kafka_brokers = test_environment.get("MLRUN_SYSTEM_TESTS_KAFKA_BROKERS") or os.getenv(
```
or it will raise an error and never hit the fallback.

Reply:
will fix